cms/service/ProxyService.py

```diff
@@ -488,6 +489,11 @@ def submission_scored(self, submission_id):
                             submission_id)
                 return
 
+            # ScoringService sent us a submission of another contest, they
```

Comment:
It is better to put this before the `hidden`/`official` checks, otherwise they will get logged for other-contest submissions as well. Same for `submission_tokened()` below.

(Following the order: from most general to least general.)